lib/virtual-fs/src/pipe.rs

```diff
@@ -213,6 +234,15 @@ impl PipeTx {
             Poll::Ready(Ok(8192))
         }
     }
+
+    fn push_readable_interest_to_rx_end(&self) {
```

Comment:
Nit: maybe `mark_read_ready` / `mark_readable`?

I know it's only an internal function, but pretty crytpic.